pkg/os/util.go

```diff
@@ -55,3 +58,30 @@ func CurrentExecutable() (string, error) {
 	}
 	return currentExec, nil
 }
+
+func CopyFileContents(src, dst string, permission os.FileMode) error {
```

Comment:
why is src not typed as string?

Reply:
In golang if 2 variable same type then each variable doesn't need explicit type definition, but I update.